larana/OpticalDetector/OpHitFinder_module.cc

```diff
@@ -94,7 +93,6 @@ namespace opdet {
     fInputModule = pset.get<std::string>("InputModule");
     fGenModule = pset.get<std::string>("GenModule");
     fInputLabels = pset.get<std::vector<std::string>>("InputLabels");
-    fUseStartTime = pset.get<bool>("UseStartTime", false);
```

Comment:
I vigorously oppose silently dropping this configuration option. Since who wrote a configuration including this parameter did expect the `absTime` to hold the start time, I believe we should let them know about the change.
A proposal is to emit a warning if the configuration requested `absTime` to hold the peak time (which the new configuration does anyway) and an exception if the configuration expressly requested `absTime` to be the start time.
Example (not compiled):
```cpp
    if (std::optional useStartTime = pset.get_if_present<bool>("UseStartTime")) {
      if (*useStartTime) {
        throw art::Exception{ art::errors::Configuration }
          << "OpHitFinder configuration option 'UseStartTime' is removed as recob::OpHit hosts both start and peak time.\n"
          << "See SBN DocDB 27124 for details.\n";
      }
      else {
        mf::LogWarning("OpHitFinder")
          << "OpHitFinder configuration option 'UseStartTime' is not supported any more (see SBN DocDB 27124).";
      }
    } // if "UseStartTime" in configuration
```
(needs `#include "messagefacility/MessageLogger/MessageLogger.h"` and `#include "canvas/Utilities/Exception.h"`, maybe something more)